librocksdb_sys/build.rs

```diff
@@ -25,6 +25,7 @@ fn main() {
     build.cpp(true).file("crocksdb/c.cc");
     if !cfg!(target_os = "windows") {
         build.flag("-std=c++11");
+        build.flag("-fno-rtti");
```

Comment:
RocksDB uses this option, if we don't use it here, the compiler will report some errors.